pkgs/top-level/factor-packages.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  pkgs,
+  overrides ? (self: super: { }),
+}:
+
+let
+  inside =
+    self:
+    let
+      callPackage = pkgs.newScope self;
+    in
+    {
```

Recursing into `factorPackags` benefits package searching and CI testing.

```suggestion
    lib.recurseIntoAttrs {
```